src/elm/Util/Time.elm

```diff
@@ -76,3 +78,36 @@ toMonthNum month =
 
         Time.Dec ->
             12
+
+
+isoStringToFullHumanized : Time.Zone -> String -> Maybe String
+isoStringToFullHumanized zone dateString =
+    case Iso8601.toTime dateString of
+        Err _ ->
+            Nothing
+
+        Ok timePosix ->
+            Just <| toFullHumanized zone timePosix
+
+
+toFullHumanized : Time.Zone -> Time.Posix -> String
+toFullHumanized zone date_ =
+    DateFormat.formatI18n DateFormat.norwegian "dd.MM.yyyy - HH:mm" zone date_
+
+
+toHoursAndMinutes : Time.Zone -> Time.Posix -> String
+toHoursAndMinutes zone date_ =
+    DateFormat.formatI18n DateFormat.norwegian "HH:mm" zone date_
+
+
+{-|
+
+    Add hours to specific posix object
```

Comment:
Denne kommentaren har blitt formattert slik pga indentering. Fjerner du den så kommer den på to linjer, sånn her:

```elm
{-| Add hours to specific posix object
-}
```